wizard.py

```diff
@@ -0,0 +1,59 @@
+import random
+import settings
+import common_fields
+import tools
+import bolus
+
+def wizard(start_time, gluc, carbs, timesteps, params):
+    wizard_data = []
+    access_settings = settings.settings(start_time, params)[0]
+    for gluc_val, carb_val, timestamp in zip(gluc, carbs, timesteps):
+        wizard_reading = {}
+        wizard_reading = common_fields.add_common_fields('wizard', wizard_reading, timestamp, params)
+        wizard_reading["bgInput"] = tools.convert_to_mmol(gluc_val)
+        wizard_reading["carbInput"] = int(carb_val)
+        wizard_reading["insulinOnBoard"] = 0
+        
+        
+        carb_ratio_sched, sensitivity_sched = access_settings["carbRatio"], access_settings["insulinSensitivity"]
+        sensitivity = tools.get_rate_from_settings(sensitivity_sched, wizard_reading["deviceTime"], "insulinSensitivity")
+        carb_ratio = tools.get_rate_from_settings(carb_ratio_sched, wizard_reading["deviceTime"], "carbRatio")
+        wizard_reading["insulinSensitivity"] = sensitivity
+        wizard_reading["insulinCarbRatio"] = carb_ratio
+        wizard_reading["bgTarget"] = { "high": access_settings["bgTarget"][0]["high"],
+                                        "low": access_settings["bgTarget"][0]["low"]}
+        wizard_reading["payload"] = {}
```

Comment:
You can just omit this. `payload` is for device-specific data, which for fake demo data almost by definition does not exist.